lib/builder.js

```diff
@@ -141,6 +155,14 @@ function addExtraOutputs(output, tree, opts) {
 Builder.prototype.buildTree = function(tree, outFile, opts) {
   var loader = this.loader;
   var self = this;
+
+  // allow passing opts
+  // as second argument.
+  if(isPlainObject(outFile)){
```

Comment:
disclosure just getting feet wet with builder but seems like new instance of Builder accepts a cfg which is of type string (loads config) or an actual config object. Am I missing something?

Regards,

Carey Hazelton
chazelton@origin1.com <mailto:chazelton@origin1.com>
Confidentiality Notice: The information contained in this electronic transmission, is privileged and confidential and is intended only for the recipient(s) named above. If the reader of this message is not the recipient(s) named above, or an authorized agent of such recipient(s) responsible for delivering it to the intended recipient(s), you are hereby notified that you have received this electronic transmission in error. Any review, dissemination, distribution, or copying of this electronic transmission including any attachments is strictly prohibited. If you have received this electronic transmission in error, please notify the sender immediately. Origin1 Technologies.


> On Apr 16, 2015, at 12:09 PM, Chris Truter <notifications@github.com> wrote:
> 
> In lib/builder.js <https://github.com/systemjs/builder/pull/142#discussion_r28540059>:
> 
> > @@ -141,6 +155,14 @@ function addExtraOutputs(output, tree, opts) {
> >  Builder.prototype.buildTree = function(tree, outFile, opts) {
> >    var loader = this.loader;
> >    var self = this;
> > +
> > +  // allow passing opts
> > +  // as second argument.
> > +  if(isPlainObject(outFile)){
> I don't believe that would be correct - when callingl new Builder(tree, null, {"real": "options}) that would take opts as {}.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/systemjs/builder/pull/142/files#r28540059>.
>